packages/build-ui/src/pages/agits/_id/management/artists/_id/page.rs

```diff
@@ -149,3 +149,241 @@ pub fn ArtistDetailPage(lang: Language, agit_id: ReadOnlySignal<i64>, artist_id:
+                                }
+                                a { class: "block px-4 py-2 text-sm text-white hover:bg-gray-700 hover:text-white",
+                                    "Delete Artist"
+                                }
+                            }
+                        }
+                    
+                }
+                            
+                    
+                        
+                    
+                    }
+                }
+
+                p { class: "text-gray-400 mb-8",
+                    "Joined Feb 20, 2025"
+                }
+
+                // Artist Info Section
+                div { class: "mb-8",
+                div { class: "flex items-center mb-4",
+                h2 { class: "text-xl font-semibold", "Artist Info" }
+                arrows::ChevronDown { class: "ml-2 [&>path]:stroke-white", height: 20, width: 20 }
+            }
+
+            // Form fields
+            div { class: "space-y-4 max-w-4xl",
+                div { class: "flex items-center",
+                    label { class: "text-sm w-40",
+                        span { class: "text-red-500 mr-1", "*" }
+                        "Display Name or Email"
+                    }
+                    input {
+                        class: "bg-border-background border border-border-primary text-white p-2.5 flex-1",
+                        placeholder: "Input",
+                        r#type: "text",
+                        value:"{ctrl.artist_input_field().display_name.clone()}",
+                        oninput: move |evt| ctrl.update_artist_field("display_name".to_string(), evt.value().clone())
+                    }
+                }
+
+                div { class: "flex items-center",
+                    label { class: "text-sm w-40", "Social Media" }
+                    input {
+                        class: "bg-border-background border border-border-primary text-white p-2.5 flex-1",
+                        placeholder: "Input",
+                        r#type: "text",
+                        value:"{ctrl.artist_input_field().social_media.clone()}",
+                        oninput: move |evt| ctrl.update_artist_field("social_media".to_string(), evt.value().clone())
+                    }
+                }
+
+                div { class: "flex items-center",
+                    label { class: "text-sm w-40",
+                        span { class: "text-red-500 mr-1", "*" }
+                        "Medium"
+                    }
+                    input {
+                        class: "bg-border-background border border-border-primary text-white p-2.5 flex-1",
+                        placeholder: "Input",
+                        r#type: "text",
+                        value:"{ctrl.artist_input_field().medium.clone()}",
+                        oninput: move |evt| ctrl.update_artist_field("medium".to_string(), evt.value().clone())
+                    }
+                }
```

Comment:
The same code is duplicated multiple times; it would be a good idea to componentize it in this way.

```suggestion
    InputField {
      label: tr.input_artist_name_label,
      required: true,
      placeholder: tr.input_artist_name_placeholder,
      value: ctrl.artist_input_field().display_name.clone(),
      oninput: move |value| ctrl.update_artist_field(value)
    }
    InputField {
      label: tr.input_social_media_label,
      required: true,
      placeholder: tr.input_social_media_placeholder,
      value: ctrl.artist_input_field().social_media.clone(),
      oninput: move |value| ctrl.update_artist_field(value)
    }
    ...
```